website/cue/reference/components/sources/base/amqp.cue

```diff
@@ -0,0 +1,240 @@
+package metadata
+
+base: components: sources: amqp: configuration: {
+	acknowledgements: {
```

Comment:
On sources IIRC we should be advising people to set it on sinks instead as it's deprecated here?

Reply:
Yeah, you're right. I'll make a note that the `AcknowledgementsConfig` type needs a better description to carry that context.